orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_runner.py

```diff
@@ -164,6 +164,13 @@ def register_context(result):
 
             # Extract results as DLPack tensors plus autograd context. Also skips all None values.
             unwrapped_values = wrap_all_outputs(result, is_training_mode)
+            ctx = unwrapped_values[0]
+            if ctx and ctx.saved_tensors:
```

Comment:
we can unpack ctx.saved_tensors just once and store as local variable.

Reply:
? but ctx.saved_tensors is only used once

Reply:
line 168, and line 169 are both triggering the unpacking operation.